workflow-templates/npm-publish.yml

```diff
@@ -3,19 +3,22 @@ on:
   push:
     branches:
       - master
+      - main
 jobs:
   release:
     name: Release
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Setup Nodejs Env
+        run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
       - name: Setup Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
-          node-version: 12
+          node-version: ${{ env.NODE_VER }}
```

Comment:
do we use the `NODE_VER` env var anywhere else? If not I think we could use `node-version-file` here instead https://github.com/actions/setup-node

Reply:
I grabbed this from https://openedx.atlassian.net/wiki/spaces/FEDX/pages/3755868171/How+to+backport+a+change+to+a+previous+version+with+semantic-release+for+JS+libraries but I like using the `node-version-file` better, I'll update.